modules/tensor_mechanics/src/materials/crystal_plasticity/CrystalPlasticityKalidindiUpdate.C

```diff
@@ -37,7 +42,13 @@ CrystalPlasticityKalidindiUpdate::CrystalPlasticityKalidindiUpdate(
     _gss_a(getParam<Real>("gss_a")),
     _ao(getParam<Real>("ao")),
     _xm(getParam<Real>("xm")),
-    _gss_initial(getParam<Real>("gss_initial"))
+    _gss_initial(getParam<Real>("gss_initial")),
+
+    _include_twinning_in_Lp(parameters.isParamSetByUser("total_twin_volume_fraction")),
+    _twin_volume_fraction_total(
+        _include_twinning_in_Lp ? &getMaterialPropertyByName<Real>(
+                                      getParam<MaterialPropertyName>("total_twin_volume_fraction"))
+                                : nullptr)
```

Comment:
why not just 
```suggestion
    _twin_volume_fraction_total(
        _include_twinning_in_Lp ? &getMaterialProperty<Real>("total_twin_volume_fraction"))
                                : nullptr)
```

Reply:
I thought we'd had a discussion in standup about how `getMaterialPropertyByName` was the better option; is that not the case? If so, I will happily make the change and update my notes

Reply:
I don't recall such a discussion. But we can talk about it tomorrow.

Reply:
correction made and notes updated